docs/developer/upgrade-guide/2-11-to-3-0.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Saleor 3.0
+---
+
+## Preparation
+
+New version introduces breaking changes and many new features. API clients like Apps and storefronts will require changes. 
+
+Before you start the process, create the backup of your data. Also let your Celery worker to process task queue before starting API migration.
+
+Due to scope of the upgrade, downtime will be required.
+
+## Multichannel
+
+Existing data will be assigned to the `"default-channel"` created during the database migration. As long, as shop has the only one channel defined, you are not required to specify the `channel` argument. Thanks to that you'll be able to migrate storefront to 3.0 granually.
+
+After adding more channels, argument will become required. Because of that, we recommend always adding the channel argument, even if you have only one channel:
+
+```graphql
+{
+  products(first: 5, channel: "default-channel") {
+    edges {
+      node {
+        id
+        name
+        description
+      }
+    }
+  }
+}
+```
+
+Besides impact on fetching the data, you'll may be required to specify channel during creating the checkouts. You'll find updated guide at [Checkout](developer/checkout.mdx) page.
+
+To read more about multichannel, check out [developer](developer/channels.mdx) and [dashboard](dashboard/channels.md) docs.
+
+### Editing channel related fields
+
+If you use API to modify objects (for example with custom App), please note that channel depending fields have dedicated mutations: updating product price require using [productChannelListingUpdate](developer/api-reference/mutations/product-channel-listing-update.mdx) mutation.
+
+Object attributes which can be defined on channel basis can be reference
+- [Collection Channel Listing](developer/api-reference/objects/collection-channel-listing.mdx)
+- [Product Channel Listing](developer/api-reference/objects/product-channel-listing.mdx)
+- [ProductVariant Channel Listing](developer/api-reference/objects/product-variant-channel-listing.mdx)
+- [Sale Channel Listing](developer/api-reference/objects/sale-channel-listing.mdx)
+- [ShippingMethod Channel Listing](developer/api-reference/objects/shipping-method-channel-listing.mdx)
+- [Voucher Channel Listing](developer/api-reference/objects/voucher-channel-listing.mdx)
+
+## Email templates
+
+New emails implementation brings easier configuration at dashboard and support for Sendgrid. If you modified default emails, you will need to migrate them to the Handlebars templates.
+
+Read documentation about user and admin email plugins:
+- [User emails](developer/available-plugins/user-emails.mdx)
+- [Admin emails plugin](developer/available-plugins/admin-emails.mdx)
+
+## Editor.js rich text editor and description fields
+
+All rich text fields now use Editor.js instead of Draft.js. Our database migration should convert old format to the new one, but visual inspection is recommended.
```

Comment:
```suggestion
All rich text fields now use [Editor.js](https://editorjs.io/) instead of [Draft.js](https://draftjs.org/). Our database migration should convert the old format to the new one, but visual inspection is recommended.
```